data/survey/6002/meta.json

```diff
@@ -0,0 +1,79 @@
+[
+    {
+        "type": "CHOICE",
+        "question": "Do you feel more knowledgable about computer networking than before you took this module?",
+        "options": [
+            "Yes",
+            "About the same",
+            "No"
+        ]
+    },
+    {
+        "type": "SCALE",
+        "question": "How helpful were the diagrams in understanding protocols?",
+        "options": [
+            "Not at all",
+            "A little",
+            "Not very",
+            "Super helpful"
+        ]
+    },
+
+    {
+        "type": "SCALE",
+        "question": "How helpful was the example of chain of command in understanding OSI layers?",
+        "options": [
+            "Not at all",
+            "A little",
+            "Not very",
+            "Super helpful"
+        ]
+    },
+    {
+        "type": "TEXT",
+        "question": "How difficult as the material to master?"
+    },
+    {
+        "type": "LONGTEXT",
+        "question": "Please list any questions you still have about the content."
+    }
+][
+    {
+        "type": "CHOICE",
+        "question": "Do you feel more knowledgable about computer networking than before you took this module?",
+        "options": [
+            "Yes",
+            "About the same",
+            "No"
+        ]
+    },
+    {
+        "type": "SCALE",
+        "question": "How helpful were the diagrams in understanding protocols?",
+        "options": [
+            "Not at all",
+            "A little",
+            "Not very",
+            "Super helpful"
+        ]
+    },
+
+    {
+        "type": "SCALE",
+        "question": "How helpful was the example of chain of command in understanding OSI layers?",
+        "options": [
+            "Not at all",
+            "A little",
+            "Not very",
+            "Super helpful"
+        ]
+    },
```

Comment:
These questions have a large jump (to my mind) of "these do not help" to "these are extremely helpful." I think the "Not very" response should be replaced with a "Somewhat" or a "Helpful" response to present a broader spectrum.